### PR TITLE
added cla and cla bot

### PR DESCRIPTION
**Pull request by Ayush7614**
This PR Contains
Added Cla 
Added Cla Bot
Added Contributor license

### CODE REVIEW

**Comment by Ayush7614** [comment]
Cc @nehagup

**Comment by Ayush7614** [comment]
cc @nehagup

**Comment by nehagup** [comment]
@Ayush7614 Please review the PR locally before raising. Seems like "LocalStack" is mentioned at multiple places.

**Comment by Ayush7614** [comment]
> @Ayush7614 Please review the PR locally before raising. Seems like "LocalStack" is mentioned at multiple places.

Done cc @nehagup

**Comment by slayerjain** [comment]
@nehagup can we merge this PR?

File: .github/workflows/cla.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        env:
+          GITHUB_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
+          PERSONAL_ACCESS_TOKEN: "${{ secrets.PRO_ACCESS_TOKEN }}"
+        with:
+          remote-organization-name: "keploy"
+          remote-repository-name: "keploy"
+          path-to-signatures: "etc/cla-signatures/signatures.json"
+          path-to-document: "https://github.com/keploy/keploy/tree/main/.github/CLA.md"
+          branch: "cla-signatures"
+          allowlist: "keploy-bot,renovate"
```
Comment (nehagup) [inline .github/workflows/cla.yml:29]: What is Keploy-bot?

**Review by nehagup — APPROVED** [review]
@Ayush7614 Approving this for now, thanks for making the PR.

**Pull request merged**